config/schema/og.schema.yml

```diff
@@ -1,5 +1,5 @@
 og.settings:
-  type: config_object
+  type: config_entity
```

Comment:
When changing to config entity the test did not broke at the beginning. Now the test fails for something else.